olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/PrefetchTilesRequest.h

```diff
@@ -203,8 +203,8 @@ class DATASERVICE_READ_API PrefetchTilesRequest final {
  private:
   std::string layer_id_;
   std::vector<geo::TileKey> tile_keys_;
-  unsigned int min_level_{0};
-  unsigned int max_level_{0};
+  unsigned int min_level_{geo::TileKey().Level()};
```

Comment:
Is it not better to add a constexpr or a alias to TileKey for this. e.g. kInvalidLevel = LevelCount.

Reply:
changed to existing geo::TileKey::LevelCount